mysite/settings.py

```diff
@@ -108,7 +108,7 @@
 
 LANGUAGE_CODE = 'en-us'

```

Comment:
好みですが、この部分を以下のようにすることで日本語を指定出来ます。
```suggestion
LANGUAGE_CODE = 'ja'
```

Reply:
把握していましたが、何となく英語の方がカッコ良さげだったので英語にしました。特にこだわりも無いのでややこしければ日本語に変更しますがどうでしょう？

Reply:
であれば、このままでも大丈夫です！